### ISSUE TITLE
Font is not detected as Mono

### ISSUE THREAD

**Issue by javlaks** [arrowtype/recursive#429]
**Problem description**
With the recent updates, the font has lost its configuration as a Mono Font. Some dev apps are not recognized as Mono

**Screenshots**
You can see how is displayed in the selector in Microsoft SQL Server Management Studio
![image](https://user-images.githubusercontent.com/513351/103221627-b1f26f00-48e8-11eb-907b-ff413c7425a1.png)

This is how this app displays for IBX Plex Mono
![image](https://user-images.githubusercontent.com/513351/103221665-c898c600-48e8-11eb-83dd-490c0887000b.png)

**Environment (please complete the following information):**
- Font Version: 1.069
- OS: Windows 10

**Comment by javlaks** [comment]
It seems to be the same than #370

**Comment by arrowtype** [comment]
Thanks for letting me know! Sorry about that.

Hmm, it might be due to the new ligatures added in 1.069. I'm sorting out another issue currently, but I'll check on this ASAP after that and hopefully provide an update to fix it within a few days.

**Comment by javlaks** [comment]
Thanks you man, and happy holidays 😉

**Comment by arrowtype** [comment]
Okay, it’s looking like it probably _is_ the same issue as before – metadata isn’t set to classify this as a monospace font.

In the `post` table, this should be `1`:

```xml
<isFixedPitch value="0"/>
```

In `OS/2`, this should be `600`:

```xml
<xAvgCharWidth value="599"/>
```

The `OS/2` Panose proption value is actually correct:

```xml
<bProportion value="9"/>
```

The `hhea` value for `advanceWidthMax` is also correct:

```xml
<advanceWidthMax value="600"/>
```

(I found these results with TTX, as well as FontBakery, which includes the following recommendations in a mono check.)

<details>
<summary><b>com.google.fonts/check/monospace</b> (Click to expand)</summary>

```
Requirements for monospace fonts:                                         
                                                                                
      * post.isFixedPitch - "Set to 0 if the font is proportionally spaced,     
      non-zero if the font is not proportionally spaced (monospaced)"           
        www.microsoft.com/typography/otspec/post.htm                            
                                                                                
      * hhea.advanceWidthMax must be correct, meaning no glyph's width value is 
      greater.                                                                  
        www.microsoft.com/typography/otspec/hhea.htm                            
                                                                                
      * OS/2.panose.bProportion must be set to 9 (monospace). Spec says: "The   
      PANOSE definition contains ten digits each of which currently describes   
      up to sixteen variations. Windows uses bFamilyType, bSerifStyle and       
      bProportion in the font mapper to determine family type. It also uses     
      bProportion to determine if the font is monospaced."                      
        www.microsoft.com/typography/otspec/os2.htm#pan                         
        monotypecom-test.monotype.de/services/pan2                              
                                                                                
      * OS/2.xAvgCharWidth must be set accurately.                              
        "OS/2.xAvgCharWidth is used when rendering monospaced fonts, at least   
      by Windows GDI"                                                           
        http://typedrawers.com/discussion/comment/15397/#Comment_15397          
                                                                                
      Also we should report an error for glyphs not of average width.           
                                                                                
      Please also note:                                                         
      Thomas Phinney told us that a few years ago (as of December 2019), if you 
      gave a font a monospace flag in Panose, Microsoft Word would ignore the   
      actual advance widths and treat it as monospaced. Source:                 
      https://typedrawers.com/discussion/comment/45140/#Comment_45140     
```

</details>

Basically, several lines are failing in my `scripts/instantiate-code-fonts.py` script (or maybe getting overwritten somehow?), resulting in default values remaining. Possibly, this is disrupted when the powerline glyphs are added ... I’ll shuffle the script to see if that is the case.

**Comment by arrowtype** [comment]
Okay, I think I got it sorted (again).

If you’d like to, could you please give the fonts in [release 1.070](https://github.com/arrowtype/recursive/releases/tag/v1.070) a try, and let me know if that solves the issue?

On my end, I’ve tested version 1.070 in Microsoft SQL Server Management Studio on Windows 10, and it seems to be working as expected:

![image](https://user-images.githubusercontent.com/45946693/103309702-c6676200-49e3-11eb-9f86-09478158d2f1.png)

**Comment by danarnold** [comment]
@arrowtype The release zip doesn't seem to have any subfolders or fonts inside of Recursive_Code.

![image](https://user-images.githubusercontent.com/1298123/103311237-ca957e80-49e7-11eb-8aad-5c4b21bdc6a5.png)

**Comment by arrowtype** [comment]
🤦‍♂️ Haha, sorry; daft mistake on my part. @danarnold can you please try it once more?

https://github.com/arrowtype/recursive/releases/tag/v1.070

**Comment by danarnold** [comment]
@arrowtype Fonts are showing as monospaced for me! I don't have the software that this issue was originally opened with so @javlaks will have to weigh in on that.

**Comment by javlaks** [comment]
Hey guys, the font is working fine again. The apps are recognizing the font as expected
![image](https://user-images.githubusercontent.com/513351/103317544-1698f100-49f1-11eb-9b33-9458fbdf5a0c.png)
![image](https://user-images.githubusercontent.com/513351/103317551-1993e180-49f1-11eb-9bb4-3caaa6531712.png)

Thank you @arrowtype 😉

**Comment by arrowtype** [comment]
Woo! Thanks for helping out on reporting and checking this, @javlaks & danarnold.
